src/app-layer-modbus.c

```diff
@@ -1580,12 +1582,14 @@ static uint8_t readCoilsRsp[] = {/* Transaction ID */    0x00, 0x00,
                                  /* Byte count */        0x03,
                                  /* Coil Status */       0xCD, 0x6B, 0x05 };
 
-static uint8_t readCoilsErrorRsp[] = {/* Transaction ID */    0x00, 0x00,
-                                      /* Protocol ID */       0x00, 0x00,
-                                      /* Length */            0x00, 0x03,
-                                      /* Unit ID */           0x00,
-                                      /* Function code */     0x81,
-                                      /* Exception code */    0x05};
+static uint8_t readCoilsErrorRsp[] = {
+    /* Transaction ID */ 0x00, 0x00,
+    /* Protocol ID */ 0x00, 0x00,
+    /* Length */ 0x00, 0x03,
+    /* Unit ID */ 0x00,
+    /* Function code */ 0x81,
+    /* Exception code */ 0xFF
```

Comment:
> readCoilsErrorRsp: changed to different invalid response code.

Is 0xFF an invalid response code ?
If so, can it appear in the comment ?

Reply:
Yes, it used to be `0x05` which is now a known exception code. We can add a comment for `0xFF`.

Reply:
> If so, can it appear in the comment ?

Do you mean in the git commit comment or as a code comment on the struct?

Reply:
a comment in the struct :
`/* Exception code */ 0xFF` looks like 0xFF is a valid exception code